packages/yoga/src/Stepper/native/Stepper.jsx

```diff
@@ -49,11 +54,14 @@ Stepper.propTypes = {
   /** Controls the active step, it receive the index value for showing some
    * step. Starting from 0. */
   activeStep: limitChildren,
+  /** Must be a color from yoga colors. */
```

Comment:
I think this description is not describing this prop since it expects a boolean. What do you think of something like:
```suggestion
  /** Use secondary color */
```